src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -145,7 +145,7 @@ public String postRequest(final int max, final int numDice, final String subject
       httpPost.addHeader("X-Triplea-Game-UUID", gameUuid);
       final String host = m_props.getProperty("host");
       final int port = Integer.parseInt(m_props.getProperty("port", "80"));
-      final HttpHost hostConfig = new HttpHost(host, port);
+      final HttpHost hostConfig = new HttpHost(host, port, "https");
```

Comment:
Doesn't have to be part of this PR, but if this is the long-term solution, we should probably make the scheme configurable via the properties file just in case we need to support dice servers that only have an HTTP endpoint.

Reply:
We could potentially take that one step further by fetching the dice server URL at the same time when we fetch the lobby url. In short I"m suggesting we add the dice server url to:
https://github.com/triplea-game/triplea/blob/master/lobby_server.yaml, then we could potentially update with immediate effect.